platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/event/orphan/OrphanBufferTests.java

```diff
@@ -206,34 +206,34 @@ private GossipEvent createRandomEvent(
     /**
      * Check if an event has been emitted or is ancient
      *
-     * @param event                 the event to check
-     * @param nonAncientEventWindow the non-ancient event window defining ancient.
+     * @param event       the event to check
+     * @param eventWindow the  event window defining ancient.
```

Comment:
unnecessary extra space between "the" and "event"

Reply:
fixed